keps/sig-cli/1802-kustomize-components/kep.yaml

```diff
@@ -8,15 +8,13 @@ owning-sig: sig-cli
 participating-sigs:
   - sig-cli
 reviewers:
-  - n/a
+  - "@pwittrock"
+  - "@monopole"
 approvers:
-  - n/a
+  - "@pwittrock"
+  - "@monopole"
 creation-date: 2020-05-20
 last-updated: 2020-05-20
-status: provisional
-see-also:
-  - n/a
-replaces:
-  - n/a
-superseded-by:
-  - n/a
+status: implemented
```

Comment:
KEPs stay as _implementable_ until they have graduated to stable. So this should move to implementable

Reply:
(ref https://github.com/kubernetes/enhancements/pull/2758#discussion_r635673334) This one is a separate KRM type that you can import from a Kustomization. It has its own API version, which is alpha, but the kustomize CLI doesn't require any flags to use it. The author already closed the tracking issue on this repo.

Reply:
Q: how does this move from provisional to implemented without passing through implementable? provisional means that it was not targeting to a release by the sig and that it wasn't officially approved for inclusion in a release.

Reply:
> Q: how does this move from provisional to implemented without passing through implementable? provisional means that it was not targeting to a release by the sig and that it wasn't officially approved for inclusion in a release.

I believe consensus was reached in the sig to move forward with this, the KEP just wasn't updated.